data/create-tables.js

```diff
@@ -14,6 +14,12 @@ client.connect()
                 task VARCHAR(256) NOT NULL,
                 completed BOOLEAN NOT NULL
             );
+            CREATE TABLE users (
```

Comment:
In order to associate a task with a specific user, you'll have to create the users table before the tasks table so that there can be a foreign key reference to the user ID in the tasks table.